### PR TITLE
clarify speed up and cancel not possbile for coinjoin or hww

### PR DESCRIPTION
**Pull request by MarnixCroes**

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -710,6 +710,8 @@ Read more [here](/using-wasabi/Send.md#speed-up-or-cancel-transaction).
 
 [![How to speed up Bitcoin transaction](https://img.youtube.com/vi/55KURak5r58/maxresdefault.jpg)](https://youtu.be/55KURak5r58)
 
+> It is not possible to speed up a coinjoin or a transaction sent with a hardware wallet.
```
Comment (MaxHillebrand) [inline docs/FAQ/FAQ-UseWasabi.md:713]: Is CPFP flow of incoming tx to hww possible?

Comment (MarnixCroes) [reply docs/FAQ/FAQ-UseWasabi.md:713]: Afaik only if you do it manually, not with the speedup tool

Comment (MaxHillebrand) [inline docs/FAQ/FAQ-UseWasabi.md:713]:
```suggestion
> It is not possible to speed up a coinjoin or a hardware wallet transaction.
```

Its for both incoming and outgoing.

File: docs/FAQ/FAQ-UseWasabi.md [hunk 2]
```diff
@@ -730,6 +732,8 @@ To cancel a transaction:
 
 Read more [here](/using-wasabi/Send.md#speed-up-or-cancel-transaction).
 
+> It is not possible to cancel a coinjoin or a transaction sent with a hardware wallet.
```
Comment (MaxHillebrand) [inline docs/FAQ/FAQ-UseWasabi.md:735]:
```suggestion
> It is not possible to cancel a coinjoin or a hardware wallet transaction.
```

**Review by MaxHillebrand — CHANGES_REQUESTED** [review]

File: docs/FAQ/FAQ-UseWasabi.md (hunk 1 above)
Comment (MarnixCroes) [reply docs/FAQ/FAQ-UseWasabi.md:713]: cack
or maybe  `The speed up tool is not available for coinjoins, or when using a hardware wallet.`

Comment (MarnixCroes) [reply docs/FAQ/FAQ-UseWasabi.md]: pushed

**Pull request merged**